src/Tasks/Microsoft.NET.Build.Tasks/GenerateBundle.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using Microsoft.NET.HostModel.Bundle;
+using System.Collections.Generic;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    public class GenerateBundle : TaskBase
+    {
+        [Required]
+        public ITaskItem[] FilesToBundle { get; set; }
+        [Required]
+        public string AppHostName { get; set; }
+        [Required]
+        public bool EmbedPDBs { get; set; }
+        [Required]
+        public string OutputDir { get; set; }
+        [Required]
+        public bool ShowDisgnosticOutput { get; set; }
+
+        protected override void ExecuteCore()
+        {
+            Bundler bundler = new Bundler(AppHostName, OutputDir, EmbedPDBs, ShowDisgnosticOutput);
```

Comment:
```suggestion
            var bundler = new Bundler(AppHostName, OutputDir, EmbedPDBs, ShowDisgnosticOutput);
```

nit: type is obvious